src/app/pages/OnDemandTvPage/index.test.jsx

```diff
@@ -300,3 +300,26 @@ it('should show the expired content message if episode is expired', async () =>
   expect(expiredMessageEl).toBeInTheDocument();
   expect(container).toMatchSnapshot();
 });
+
+it('should show the future content message if episode is not yet available', async () => {
```

Comment:
Worth chucking in a test for the 'pending' availability too to confirm it renders the correct `notYetAvailable` message?

Also maybe the variable here should be renamed `notYetAvailableMessage` rather than `expiredMessage`?